components/ruby/bundler/Makefile

```diff
@@ -0,0 +1,58 @@
+
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright 2016 Grüninger GmbH, Andreas Grüninger (grueni)
+#
+include ../../../make-rules/shared-macros.mk
+
+RUBY_VERSION = 2.2
+RUBY_LIB_VERSION = 2.2.0
```

Comment:
RUBY_LIB_VERSION, RUBY.2.2 and RUBY definitions should be moved to shared-macros.mk, as they are useful for other projects as well. I don't consider it a blocker. @pyhalov should we move these?